diff --git external/hash/initbuild.sh external/hash/initbuild.sh
index 91117a6..cb1fc0b 100755
--- external/hash/initbuild.sh
+++ external/hash/initbuild.sh
@@ -2,4 +2,4 @@
 
 cd `dirname $0`
 mkdir -p "build"
-cd build && cmake -GNinja .. && ninja
+cd build && cmake ${XTOOLFILE} .. && make
diff --git scripts/build.sh scripts/build.sh
index 7cce9d7..ea5bfc5 100755
--- scripts/build.sh
+++ scripts/build.sh
@@ -17,7 +17,7 @@ if [ ! -d ${ROOT}/build/Debug ] || [ ! -d  ${ROOT}/build/Release ]; then
 fi
 
 echo "building Debug" 1>&2
-cd ${ROOT}/build/Debug && ninja
+cd ${ROOT}/build/Debug && make
 
 echo "building Release" 1>&2
-cd ${ROOT}/build/Release && ninja
+cd ${ROOT}/build/Release && make
diff --git scripts/initbuild.sh scripts/initbuild.sh
index 1700ca5..34836b1 100755
--- scripts/initbuild.sh
+++ scripts/initbuild.sh
@@ -2,7 +2,7 @@
 
 set -e
 
-echo "initializing build for CMake Ninja"
+echo "initializing build for CMake Makefiles"
 
 cd `dirname $0`/..
 ROOT=`pwd`
@@ -12,5 +12,5 @@ mkdir -p ${ROOT}/build/Release
 rm -rf ${ROOT}/build/Debug/*
 rm -rf ${ROOT}/build/Debug/*
 
-cd ${ROOT}/build/Debug && cmake -GNinja ../.. -DCMAKE_BUILD_TYPE=Debug
-cd ${ROOT}/build/Release && cmake -GNinja ../.. -DCMAKE_BUILD_TYPE=Release
+cd ${ROOT}/build/Debug && cmake ../.. -DCMAKE_BUILD_TYPE=Debug ${XTOOLFILE}
+cd ${ROOT}/build/Release && cmake ../.. -DCMAKE_BUILD_TYPE=Release ${XTOOLFILE}
